### ISSUE TITLE
bugous readme file example

### ISSUE THREAD

**Issue by inf3rno** [cucumber/cucumber-js#344]
I tried to use this lib on win7x64 with webstorm. I used the default cucumber test settings with `D:\creation\software developer\projects\scheduler\features` as feature directory.

I added a simple self-testing code to the feature directory to verify the installation:

cucumber.feature
```cucumber
Feature: Environment self-test
  As a developer
  In Order to verify whether the testing environment is working properly
  I want to write a simple test, which does not fail

  Scenario: Simple test
    When the testing environment is working properly
    Then this test should not fail or throw any errors
```

support/cucumber.world.js
```js
//var log = require("../../log");

module.exports = function () {
    //log("world");
    this.World = function World(callback) {
        //log("world constructor");
        this.worldSpecificMethod = function (i) {
            //log("world specific method");
            return i + 1;
        };
        //log("world constructor end");
        callback();
    };
    //log("world end");
};
```

step_definitions/cucumber.js
```js
var assert = require("assert");
//var log = require("../../log");

module.exports = function () {
    //log("step definitions");

    this.World = require("../support/cucumber.world").World;

    this.When(/^the testing environment is working properly$/, function (callback) {
        //log("when");
        this.i = this.worldSpecificMethod(12);
        //log("when end");
        callback();
    });

    this.Then(/^this test should not fail or throw any errors$/, function (callback) {
        //log("then");
        assert.equal(this.i, 13);
        //log("then end");
        callback();
    });

    //log("step definitions end");
};
```

webstorm console
```
C:\Users\inf3rno\AppData\Roaming\npm\cucumber-js.cmd -format=summary --require "D:/creation/software developer/projects/scheduler/features" --require "C:/Program Files (x86)/JetBrains/WebStorm 9.0.3/plugins/CucumberJavaScript/lib/cucumberjs_formatter_windows.js"
Testing started at 0:03 ...


events.js:72
        throw er; // Unhandled 'error' event
              ^
TypeError: undefined is not a function
  at C:\Program Files (x86)\JetBrains\WebStorm 9.0.3\plugins\CucumberJavaScript\lib\cucumberjs_formatter_windows.js:83:9
  at handleBeforeFeaturesEvent (C:\Program Files (x86)\JetBrains\WebStorm 9.0.3\plugins\CucumberJavaScript\lib\cucumberjs_formatter_windows.js:71:9)
  at Object.<anonymous> (C:\Users\inf3rno\AppData\Roaming\npm\node_modules\cucumber\bin\cucumber.js:4:5)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Function.Module.runMain (module.js:497:10)
  at startup (node.js:119:16)
  at node.js:929:3

```

log (it runs twice for some reason and the world constructor and the step definitions never run)
```
world
world end
step definitions
step definitions end

world
world end
step definitions
step definitions end
```
Any idea how to fix it?

off: I hate libs which are hard to install and hard to debug (because of the lack of proper error messages)... :S

**Comment by inf3rno** [comment]
I upgraded from node 0.10 to 0.12, I got somewhat different error message now

```
C:\Users\inf3rno\AppData\Roaming\npm\cucumber-js.cmd -format=summary --require "D:/creation/software developer/projects/scheduler/features" --require "C:/Program Files (x86)/JetBrains/WebStorm 9.0.3/plugins/CucumberJavaScript/lib/cucumberjs_formatter_windows.js"
Testing started at 0:53 ...

C:\Users\inf3rno\AppData\Roaming\npm\node_modules\cucumber\lib\cucumber\support_code\library.js:109
      var world = new WorldConstructor(function (explicitWorld) {
                  ^
TypeError: undefined is not a function
  at C:\Program Files (x86)\JetBrains\WebStorm 9.0.3\plugins\CucumberJavaScript\lib\cucumberjs_formatter_windows.js:83:9
  at handleBeforeFeaturesEvent (C:\Program Files (x86)\JetBrains\WebStorm 9.0.3\plugins\CucumberJavaScript\lib\cucumberjs_formatter_windows.js:71:9)
  at Object.<anonymous> (C:\Users\inf3rno\AppData\Roaming\npm\node_modules\cucumber\bin\cucumber.js:4:5)
  at Module._compile (module.js:460:26)
  at Object.Module._extensions..js (module.js:478:10)
  at Module.load (module.js:355:32)
  at Function.Module._load (module.js:310:12)
  at Function.Module.runMain (module.js:501:10)
  at startup (node.js:129:16)
  at node.js:814:3
```

**Comment by inf3rno** [comment]
This fixed the problem:
```js
module.exports = {
    World : function World(callback) {
        //log("world constructor");
        this.worldSpecificMethod = function (i) {
            //log("world specific method");
            return i + 1;
        };
        //log("world constructor end");
        callback();
    }
};
```

which is interesting, since I got the example from the readme file:

```js
// features/support/world.js
module.exports = function () {
  var zombie = require('zombie');
  this.World = function World(callback) {
    this.browser = new zombie(); // this.browser will be available in step definitions

    this.visit = function (url, callback) {
      this.browser.visit(url, callback);
    };

    callback(); // tell Cucumber we're finished and to use 'this' as the world instance
  };
}
```

probably the readme file is bugous

**Comment by jbpros** [comment]
Thanks @inf3rno, it was indeed a mistake in the README.

Fixed as of 46a7411
